packages/components/src/menu-item/style.scss

```diff
@@ -2,6 +2,13 @@
 .components-menu-item__button.components-button {
 	width: 100%;
 
+	&[role="menuitemradio"],
+	&[role="menuitemcheckbox"] {
+		.components-menu-item__item:only-child {
+			padding-right: $grid-unit-60;
```

Comment:
Should we add a comment to denote what this particular value does?

Reply:
Added 👍